deepmd/pt/entrypoints/main.py

```diff
@@ -319,6 +322,20 @@ def main(args: Optional[Union[List[str], argparse.Namespace]] = None):
             FLAGS.model = FLAGS.checkpoint_folder
         FLAGS.output = str(Path(FLAGS.output).with_suffix(".pth"))
         freeze(FLAGS)
+    elif FLAGS.command == "show":
```

Comment:
better to pack the implementation in a function, and provide a UT for it.

Reply:
Done